pkg/api/api.go

```diff
@@ -38,6 +40,7 @@ func New(profileCatalog Catalog) API {
 	r.HandleFunc("/profiles", a.ProfilesHandler)
 	r.HandleFunc("/profiles/{catalog}/{profile}", a.ProfileHandler)
 	r.HandleFunc("/profiles/{catalog}/{profile}/{version}", a.ProfileWithVersionHandler)
+	r.HandleFunc("/profiles/{catalog}/{profile}/{version}/update", a.ProfileGreaterThanVersionHandler)
```

Comment:
Sorry in advance D: 

I know we are picking one and running, but `update` just seems confusing, mainly from a CRUD point of view too. Wb `newer` or `available_updates` or something that doesn't make me think its updating the profile :joy: . It reads slightly cleaner:
```
/profiles/nginx-catalog/weaveworks-nginx/v0.1.0/update
vs
/profiles/nginx-catalog/weaveworks-nginx/v0.1.0/newer
/profiles/nginx-catalog/weaveworks-nginx/v0.1.0/newer_versions
/profiles/nginx-catalog/weaveworks-nginx/v0.1.0/available_updates

```